fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2171,7 +2173,38 @@ public void testDeleteLargeLiteralStoredAsBinary() throws IOException {
     }
 
     @Test
-    public void testCreationResponseDefault() {
+    public void testInboundLinksDoNotUpdateEtag() throws IOException {
+        final String id1 = getRandomUniqueId();
+        final HttpPut httpPut = putObjMethod(id1);
+        final String oldETag;
+        final String oldMod;
+        try (final CloseableHttpResponse response = execute(httpPut)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+            oldETag = response.getFirstHeader("ETag").getValue();
+            oldMod = response.getFirstHeader("Last-Modified").getValue();
+        }
+
+        final String id2 = getRandomUniqueId();
+        createObject(id2).close();
+
+        final HttpPatch patch = patchObjMethod(id2);
+        patch.addHeader("Content-Type", "application/sparql-update");
+        patch.setEntity(new StringEntity(
+                "INSERT { <> <http://purl.org/dc/elements/1.1/relation> <" + serverAddress + id1 + "> } WHERE {}"));
+        try (final CloseableHttpResponse response = execute(patch)) {
```

Comment:
Isn't this just [this](https://github.com/acoburn/fcrepo4/blob/b456ebc624ce73dd358abc2acd7d1c783c71e16d/fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/AbstractResourceIT.java#L225)?